torch/_dynamo/utils.py

```diff
@@ -995,6 +995,24 @@ def safe_str(item: Any) -> str:
         return ",".join(safe_str(item) for item in metric)
 
     structured_logging_overhead_s = torch._logging.get_structured_logging_overhead()
+
+    if torch._inductor.utils.should_use_remote_fx_graph_cache():
```

Comment:
I saw a comment from ezyang on the internal diff that he'd prefer if we centralize this logic so it's less likely to get out of sync?